crates/fluvio-stream-dispatcher/src/store/mod.rs

```diff
@@ -32,16 +35,16 @@ mod context {
 
     pub type StoreChanges<S, MetaContext = K8MetaItem> = MetadataChanges<S, MetaContext>;
 
-    static MAX_WAIT_TIME: Lazy<u64> = Lazy::new(|| {
+    static MAX_WAIT_TIME: Lazy<Option<u64>> = Lazy::new(|| {
         use std::env;
 
         let var_value = env::var("FLV_DISPATCHER_WAIT").unwrap_or_default();
-        let wait_time: u64 = var_value.parse().unwrap_or(10);
+        let wait_time: Option<u64> = var_value.parse().ok();
         wait_time
     });
 
     #[derive(Debug, Clone)]
-    pub struct StoreContext<S, MetaContext = K8MetaItem>
+    pub struct StoreContext<S, MetaContext = K8MetaItem, const DEFAULT_WAIT_TIME: u64 = 10>
```

Comment:
These changes add more generic arg, which makes it more complex and slows down compile time.

Reply:
I added that generic const, just because wait time for `fluvio` `StoreContext` is 60 seconds and for `fluvio-stream-dispatcher` is 10 seconds. Not sure which other approach we could do for that.